site/src/comparison.rs

```diff
@@ -1314,11 +1323,13 @@ mod tests {
                 (Category::Primary, 1.0, 3.0),
             ],
             r#"
-|            | Regressions 😿 <br />(primary) | Regressions 😿 <br />(secondary) | Improvements 🎉 <br />(primary) | Improvements 🎉 <br />(secondary) | All 😿 🎉 <br />(primary) |
-|:----------:|:------------------------------:|:--------------------------------:|:-------------------------------:|:---------------------------------:|:------------------------:|
-| count[^1]  | 3                              | 0                                | 0                               | 0                                 | 3                        |
-| mean[^2]   | 146.7%                         | N/A                              | N/A                             | N/A                               | 146.7%                   |
-| max        | 200.0%                         | N/A                              | N/A                             | N/A                               | 200.0%                   |
+|            | mean[^1] | max | count[^2] |
+|:----------:|:--------:|:---:|:---------:|
+| Regressions 😿 <br /> (primary) | 146.7%   | 200.0% | 3         |
```

Comment:
TBH, I'm reluctant to optimize for plain text given how frequently is the table changing (and that it will also probably change again in the near future). But if you consider it important, I'll just set a fixed width for the columns.

Reply:
https://docs.rs/tabled/latest/tabled/ may be worth considering in the future as a dependency to ease our own work to build these.